CONTRIBUTING.md

```diff
@@ -2,29 +2,31 @@
 
 ## Feedback
 
-The report of issues and the proposal of new features is very welcome.
-Please use the GitHub issue page for this.
+Issue reports and feature proposals are very welcome.
+Please use the [GitHub issue page](https://github.com/glotzerlab/signac-flow/issues/) for this.
 
 ## Contributing code
 
-Code contributions to the signac-flow open-source project are welcomed via pull requests on GitHub.
-Prior any work you should contact the signac-flow developers to ensure that the planned development meshes well with the directions and standards of the project.
+Code contributions to the signac open-source project are welcomed via pull requests on GitHub.
+Prior any work you should contact the signac developers to ensure that the planned development meshes well with the directions and standards of the project.
 All contributors must agree to the Contributor Agreement ([ContributorAgreement.md](ContributorAgreement.md)) before their pull request can be merged.
 
-General guidelines:
+### General guidelines
 
-  * Use the [git flow][git flow] model of development:
-    - New features should be developed in a feature branch based on `develop`.
+  * Use the [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) model of development:
```

Comment:
This description of "git flow" is accurate for the model of development we've been using, except for possibly release branches, right? (re: potentially changing to "OneFlow" conversation in Slack)

Reply:
I did not touch this. We should clarify our branching model and then updates the guidelines on all affected repositories.